### ISSUE TITLE
Mesh structure problem, discontinuity

### ISSUE THREAD

**Issue by MacCCLAn** [AiuniAI/Unique3D#3]
This is really a wonderful work, And test found that the normal map prediction of some parts will be chaotic.
there are obvious defects in the mesh space, and suddenly a part will grow out or other.
![20240602111531](https://github.com/AiuniAI/Unique3D/assets/157973039/9bebfbc8-d91d-4aea-ac01-ed5c845152c8)
Is there any progress in the optimization of this part?

**Comment by wukailu** [comment]
Yes, we have observed this as well. The main source of this problem is caused by the use of only normal supervision. We are considering how to address this issue.

**Comment by ChengMacsaewe** [comment]
> Yes, we have observed this as well. The main source of this problem is caused by the use of only normal supervision. We are considering how to address this issue.

OK，Great! Thanks for your attention, Looking forward to the next edition of the model！！

When open the finetune training code?

**Comment by MrForExample** [comment]
> Yes, we have observed this as well. The main source of this problem is caused by the use of only normal supervision. We are considering how to address this issue.

Maybe add some sort of depth supervision?

**Comment by bc129697** [comment]
How do you run this demo?   use this "python app/gradio_local.py --port 7860" ?

**Comment by bc129697** [comment]
> This is really a wonderful work, And test found that the normal map prediction of some parts will be chaotic. there are obvious defects in the mesh space, and suddenly a part will grow out or other. ![20240602111531](https://private-user-images.githubusercontent.com/157973039/335838074-9bebfbc8-d91d-4aea-ac01-ed5c845152c8.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk5MDM4MTMsIm5iZiI6MTcxOTkwMzUxMywicGF0aCI6Ii8xNTc5NzMwMzkvMzM1ODM4MDc0LTliZWJmYmM4LWQ5MWQtNGFlYS1hYzAxLWVkNWM4NDUxNTJjOC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwMlQwNjU4MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZDRiY2RjYTVkODFkYTU1OTU2ZDI1MjRiZTc2NjdhYjFkYWI2YTcyM2FmOWY5YWE4MDYyZWJkNmE3MTMxNmI0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ooDbRaDszj0wyEeilQxyxgsit_2Id0tGdDXe6Kaey_w) Is there any progress in the optimization of this part?

How do you run this demo?   use this "python app/gradio_local.py --port 7860" ?
